pkg/products/bamboo/variables.tf

```diff
@@ -50,4 +50,34 @@ variable "db_instance_class" {
 variable "db_iops" {
   description = "The requested number of I/O operations per second that the DB instance can support."
   type        = number
+}
+
+variable "license" {
+  description = "Bamboo license."
+  type        = string
+  sensitive   = true
+}
+
+variable "admin_username" {
```

Comment:
Here the only sensitive items are license and admin password. All other variables are normal and could be seen as plain text.

Reply:
yea, I'm not too sure about this part either. I'm basically open to either way. 

The reason I'm marking it as sensitive is because all of them are stored in Kubernetes secrets. And I guess whether it's sensitive or nor is not just about the nature of the value itself, but how we implement/store it, in this case, as a Kubernetes secret. 

@nanux @jjeongatl What's your opinion?

Reply:
I would make just password sensitive.

Reply:
Cool~ insensitive it is then.